scripts/linters/custom_eslint_checks/rules/no-multiline-disable.js

```diff
@@ -0,0 +1,64 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Lint check to ensure that there is a break after parenthesis
+ * in case of multiline hanging indentation.
+ */
+
+'use strict';
+
+module.exports = {
+  meta: {
+    type: 'layout',
+    docs: {
+      description: (
+        'Lint check to ensure that there is a break after parenthesis in case' +
+        ' of multiline hanging indentation'),
```

Comment:
Hmmm.. I think this is copy-pasta :p fix this and do a self-review of this PR. :) (It helps you get your PR merged quickly.)